### DISCUSSION TITLE
How to send message?

### DISCUSSION THREAD

**Discussion by Eltar007** [egramtel/tdsharp#142] [Q&A] [answered]
Can you help me to send message via this method
`await _client.ExecuteAsync(new TdApi.SendMessage()
        {
            ChatId = 832378177,
            DataType = "messageText",
            InputMessageContent = content
        });`
How is it possible to create InputMessageContent correctly?

**Comment by Eltar007** [comment, marked as answer]
I've received this task )
`// Getting chats to get access to my chat
        var chats = await _client.ExecuteAsync(new TdApi.GetChats()
        {
            Limit = 500
        });
        // compare content
        TdApi.InputMessageContent content = new TdApi.InputMessageContent.InputMessageText()
        {
            Extra = null,
            Text = new TdApi.FormattedText()
            {
                Extra = null,
                Text = "hello world!"
            }
        };
        // sending compared content
        await _client.ExecuteAsync(new TdApi.SendMessage()
        {
            ChatId = 832378177,
            InputMessageContent = content
        });`

![image](https://user-images.githubusercontent.com/47949652/187673716-54fcb0c4-3256-4a2e-8363-25debcd65f3e.png)

**Reply by ForNeVeR** [reply]
So, you've figured it out yourself? The code looks correct.
